workspace/assets/css/lib/_reset.less

```diff
@@ -68,6 +68,11 @@ button {
 	background: none;
 }
 
+fieldset {
+	border: 0px;
+	padding: 0px
+}
+
```

Comment:
I think we should merge `fieldset`, `blockquote`, and `figure` with our "html5" declaration...